content/resources/operator/_index.md

```diff
@@ -81,7 +81,106 @@ Blood Pressure Reading:
+
+_Pros_
+
+* User’s organization inherently takes responsibility for actions taken by user under the auspice of their organizational identity.
+* User’s organization takes responsibility for activating/deactivating and monitoring identity.
+
+_Cons_
+
+* Can only be used if the user’s organization utilizes Google for identity management.
+
+### Microsoft Office 365 (organizational account)
+
+Microsoft Office 365 secure authentication system as provided by the user’s organization.
+
+_Pros_
+
+* User’s organization inherently takes responsibility for actions taken by user under the auspice of their organizational identity.
+* User’s organization takes responsibility for activating/deactivating and monitoring identity.
+
+_Cons_
+
+* Can only be used if the user’s organization utilizes Microsoft Office 365 for identity management.
+
+### InCommon
+
+InCommon, operated by Internet2, provides a secure and privacy-preserving trust fabric for research and higher education, and their partners, in the United States. Individual identity providers, such as NIH iTrust and most academic institutions, are federated by InCommon.
+
+_Pros_
+
+* User’s organization inherently takes responsibility for actions taken by user under the auspice of their organizational identity.
+* User’s organization takes responsibility for activating/deactivating and monitoring identity.
+
+_Cons_
+
+* Can only be used if the user’s organization is part of the InCommon federation. 
+* Reference this resource for organizations supported by InCommon.
+
+### eduGAIN
+
+eduGAIN is an international “interfederation” of identity and service providers around the world. InCommon is a participant in eduGAIN.
+
+_Pros_
+
+* User’s organization inherently takes responsibility for actions taken by user under the auspice of their organizational identity.
+* User’s organization takes responsibility for activating/deactivating and monitoring identity.
+* International presence, and InCommon is a participant
+
+_Cons_
+
+* Can only be used if the user’s organization is part of one of the 60+ federations participating in eduGAIN. 
+* Reference this resource for organizations supported by eduGAIN.
+
+### ORCID
+
+Provide an identifier for individuals to use with their name as they engage in research, scholarship, and innovation activities. 
+
+_Pros_
+
+* Most (all) researchers either have an ORCID or can easily create an ORCID.
+
+_Cons_
+
+* User’s organization does not inherently take responsibility for actions taken by user under auspice of ORCID identity.
+* Neither user’s organization nor ORCID take responsibility for activating/deactivating users based on affiliation.
+
+### Google / Microsoft Office 365 (personal account)
+
+Google Sign-In / Microsoft Office 365 secure authentication systems as registered by an individual with no organizational affiliation or management.
+
+_Pros_
+
+* Most users either have an Google/Microsoft identities or can easily create one.
```

Comment:
...have a ... identity...
...have Google/Microsoft identities ...

Reply:
Thanks! Done in 3d1dc98.